Fix installing firefox in browser tests

### Describe the bug

Current browser tests CI installs Firefox by running https://github.com/ChainSafe/lodestar/blob/3f3c7fcb3476d66550515958a4b80e81b07ea87b/.github/workflows/test.yml#L181


However it attempts to download a browser by pointing to an url that no longer exists:
```
Run npx @puppeteer/browsers install firefox@latest --path /tmp
@puppeteer/browsers install <browser>

Download and install the specified browser. If successful, the command outputs the actual browser buildId that was installed and the absolute path to the browser executable (format: <browser>@<buildID> <path>).
...
Error: Download failed: server returned code 404. URL: https://archive.mozilla.org/pub/firefox/nightly/latest-mozilla-central/firefox-135.0a1.en-US.linux-x86_64.tar.bz2
    at /home/runner/actions-runner/_work/lodestar/lodestar/node_modules/@puppeteer/browsers/lib/cjs/httpUtil.js:95:31
    at ClientRequest.requestCallback (/home/runner/actions-runner/_work/lodestar/lodestar/node_modules/@puppeteer/browsers/lib/cjs/httpUtil.js:72:13)
    at Object.onceWrapper (node:events:635:26)
    at ClientRequest.emit (node:events:520:28)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (node:_http_client:700:27)
    at HTTPParser.parserOnHeadersComplete (node:_http_common:119:17)
    at TLSSocket.socketOnData (node:_http_client:542:22)
    at TLSSocket.emit (node:events:520:28)
    at addChunk (node:internal/streams/readable:559:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)
Error: Process completed with exit code 1.
```

Hence failing the entire browser tests

### Expected behavior

Successfully pass the browser tests

### Steps to reproduce

_No response_

### Additional context

_No response_

### Operating system

Linux

### Lodestar version or commit hash

v1.24.0

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -178,7 +178,7 @@ jobs:
       - name: Install Chrome browser
         run: npx @puppeteer/browsers install chromedriver@latest --path /tmp
       - name: Install Firefox browser
-        run: npx @puppeteer/browsers install firefox@latest --path /tmp
+        run: npx @puppeteer/browsers install firefox@stable --path /tmp
       - name: Browser tests
         run: |
           export DISPLAY=':99.0'

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -176,7 +176,7 @@ jobs:
         with:
           node: ${{ matrix.node }}
       - name: Install Chrome browser
-        run: npx @puppeteer/browsers install chromedriver@latest --path /tmp
+        run: npx @puppeteer/browsers install chromedriver@stable --path /tmp
       - name: Install Firefox browser
         run: npx @puppeteer/browsers install firefox@stable --path /tmp
       - name: Browser tests

--- a/vitest.base.browser.config.ts
+++ b/vitest.base.browser.config.ts
@@ -51,7 +51,7 @@ export default defineConfig({
       screenshotFailures: false,
       providerOptions: {
         capabilities: {
-          browserVersion: "latest",
+          browserVersion: "stable",
         },
       },
     },

